silx/gui/plot/backends/BackendBase.py

```diff
@@ -263,7 +258,7 @@ def addMarker(self, x, y, legend, text, color,
                           and that returns the filtered coordinates.
         :return: Handle used by the backend to univocally access the marker
         """
-        return legend
+        return object()
```

Comment:
What about `raise NotImplementedError()`, `BaseBackend` is an abstract class no? It have to be implemented.

Reply:
The `BackendBase` can be use to run the `PlotWidget` headless. This is done in the tests at some point.
Basically all methods of `BackendBase` should raise `NotImplementedError`.
Doing it this way provides a mock and the definition of the interface at once.

Reply:
Ok.